### ISSUE TITLE
Error in example twig

### ISSUE THREAD

**Issue by henrypenny** [KnpLabs/KnpLastTweetsBundle#30]
I'm currently trying to test the following code:

    {% extends 'AppBundle::base.html.twig' %}
    
    {% block body %}
        {% render "KnpLastTweetsBundle:Twitter:lastTweets" with {'username': 'knplabs'} %}
    {% endblock body %}

This is showing the following error:

    Unexpected token "name" of value "with" ("end of statement block" expected) in src/AppBundle/Resources/views/Default/index.html.twig at line 4

**Comment by weaverryan** [comment]
Yep, the code example in the readme's uses the old render dynasty - we should update that.
